src/main/java/com/autotune/common/data/result/ExperimentResultData.java

```diff
@@ -31,9 +31,9 @@ public class ExperimentResultData {
     private String experiment_name;
     private String trialNumber;
     @SerializedName("interval_start_time")
```

Comment:
I think this also can be changed to it's corresponding constant value similar to how it's done in the IntervalResults class

Reply:
done, thanks